src/index.html

```diff
@@ -32,8 +32,7 @@ <h3>Migrate an existing <br />Zigbee network</h3>
     </div>
     <div>
       <h3>Thread and Matter</h3>
-      <p> A future firmware update
-        will bring Thread support; allowing SkyConnect to power your Matter and
-        Zigbee networks at the same time.</p>
+      <p> SkyConnect comes with experimental support to power Matter and
+        Zigbee networks at the same time. Before [enabling multiprotocol support](/procedures/enable-multiprotocol/), check the section about [multiprotocol issues](about-multiprotocol).</p>
```

Comment:
This is (and was) not ideally worded.

```suggestion
      <p>SkyConnect comes with experimental support to power Thread (used for some Matter devices) and
        Zigbee networks at the same time. Before [enabling multiprotocol support](/procedures/enable-multiprotocol/), check the section about [multiprotocol issues](about-multiprotocol).</p>
```